maxbytes_go118.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !go1.19
+
+package connect
+
+import "strings"
+
+func asMaxBytesError(situation string, err error) *Error {
+	const expect = "http: request body too large"
+	text := err.Error()
```

Comment:
Any benefit to using errors.Is here instead or will we never expect it to be wrapped?

Reply:
We can't use `errors.Is` on Go 1.18 - the `http.MaxByteError` type was introduced in Go 1.19.